server/routes/resources.ts

```diff
@@ -0,0 +1,53 @@
+import * as express from 'express';
+import  {Request, Response } from 'express'
```

Comment:
These express imports should probably be merged to online, like: `import express, { Request, Response } from 'express';`

Reply:
Fixed it. Thanks